File: benchmarks/bench-nitro/ios/HybridBench.swift
import NitroModules

class HybridBench: HybridBenchSpec {
  func addNumbers(a: Double, b: Double) throws -> Double {
    return a + b
  }

  func addStrings(a: String, b: String) throws -> String {
    return a + b
  }
}

File: benchmarks/bench-nitro/android/src/main/java/com/margelo/nitro/benchnitro/HybridBench.kt
package com.margelo.nitro.benchnitro

import com.facebook.proguard.annotations.DoNotStrip

@DoNotStrip
class HybridBench : HybridBenchSpec() {
  override fun addNumbers(a: Double, b: Double): Double = a + b

  override fun addStrings(a: String, b: String): String = a + b
}
